components/Card.jsx

```diff
@@ -130,3 +128,22 @@ const StyledButton = styled.button`
   width: var(--button-width);
   border-radius: var(--button-border-radius);
 `;
+const StyledPopup = styled.div`
+  position: relative;
+  text-align: center;
+  height: 70vw;
+  width: 70vw;
+  position: relative;
+  background-color: white;
+  border: var(--card-border);
+  margin-bottom: 10rem;
+  & button {
```

Comment:
i think you can get rid of the &
```suggestion
  button {
```